docs/samples/triton/bert/bert.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: "serving.kubeflow.org/v1alpha2"
+kind: "InferenceService"
+metadata:
+  name: "bert-large"
+spec:
+  default:
+    transformer:
+      custom:
+        container:
+          name: kfserving-container      
+          image: gcr.io/kubeflow-ci/kfserving/bert-transformer:latest
+          resources:
+            limits:
+              cpu: "1" 
+              memory: 1Gi
+            requests:
+              cpu: "1" 
+              memory: 1Gi
+          command:
+            - "python"
+            - "-m"
+            - "bert_transformer"
+          env:
+            - name: STORAGE_URI
```

Comment:
Unrelated question but is there a good reason this isn't just part of the spec like for most predictors?

Reply:
I would love to promote some transformers to first class in the spec. Take a look at the feast transformer PR I wrote 6 months ago.

Reply:
I was referring to storageUri in the spec vs STORAGE_URI. I read your comment as bundling common transformers? I like that idea too, but doesn’t necessarily have to be part of the API spec.

Reply:
Oh yeah oops. Definitely misunderstood your question :). 

Transformers don't have a concept of storage uri (it's just a pure customspec), but maybe it makes sense to in the v1alpha3 as a way of interacting with the storage-initializer.

Reply:
I’d suggest that for the customSpec. We do a customSpecs and always mount in storage as well to load the models. Building different docker images every time we want to update a model seems like a huge waste.